hack/community/generate.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# Generate the new community manifests, from the current WMCO manifests.
+
+# Must be ran within WMCO root.
+
+# Example:
+# Cut community v5.1.0 for community-4.10
+# Run: bash ./hack/community/generate.sh 5.1.0 community-4.10
+
+clone_community() {
+  local CO_GIT="https://github.com/${CO_REPO}.git"
+
+  git clone "${CO_FORK}" "${CO_DIR}"
+  pushd "${CO_DIR}"
+  git remote add upstream "${CO_GIT}"
+  git fetch upstream main:upstream/main
+  git checkout upstream/main
+  git config user.name "${GITHUB_NAME}"
+  git config user.email "${GITHUB_EMAIL}"
+  echo "   ${CO_PROJECT} cloned"
+  popd
+}
+
+replace() {
+    CREATED_AT=$(date +"%Y-%m-%dT%H:%M:%SZ")
+    VALUE="community-windows-machine-config-operator"
+    NAME="${VALUE}-${OPERATOR_VERSION}"
+    DISPLAY_NAME="Community Windows Machine Config Operator"
+    MATURITY="preview"
+    QUAY_IMAGE=
+    DESCRIPTION=$(cat hack/community/csv/description.md)
+
+    /tmp/yq eval --exit-status --inplace '
```

Comment:
Given `yq` is a [requirement](https://github.com/openshift/windows-machine-config-operator/pull/1050/files#diff-9d3c5e6a1d34d7b8500f674dd3595ec1a6ab6ebbed5dada1fef462fc589d12bbR86),  `/tmp/yq` may not work. I suggest to using the existing `yq` from  `$PATH`.